Microsoft.Toolkit.Uwp.UI/Extensions/Tree/LogicalTree.cs

```diff
@@ -200,7 +200,8 @@ public static IEnumerable<T> FindChildren<T>(this FrameworkElement element)
         }
 
         /// <summary>
-        /// Finds the logical parent element with the given name or returns null.
+        /// Finds the logical parent element with the given name or returns null. Note: Parent may only be set when the control is added to the VisualTree.
```

Comment:
Is it worth noting that this might not be able to resolve something if we are not in the visual tree, which is not a common scenario. However controls such as TeachingTip or MenuFlyout are commonly created in code behind and using parent helpers on them might return faulty results.

Reply:
Any suggestion for better text here then?

Reply:
Turns out, I completely overread the part with parent, sorry about that. Would it be worth adding that to the description of the LogicalTree class?

Reply:
Oh, maybe vs. on the method here specifically? Or on both?

Reply:
That's a good question. I think that something like this most likely would be pointed out on a remarks section. I think often it is being called out on the functions and the class but not sure.

Reply:
@michael-hawker I think we can do this in another PR so we can merge this and unblock @jamesmcroft !